src/eei.h

```diff
@@ -75,6 +75,37 @@ struct EthereumInterface : ShellExternalInterface {
     }
   }
 
+  void memRead(uint32_t offset, uint8_t *dst, size_t length)
+  {
+     uint32_t i;
+     for (i = offset; i < (offset + length); i++) {
```

Comment:
Type declaration could be folded into the for loop, though this hear is susceptible to overflow (size requirement of offset + length might be bigger than the size of i)